src/java/fr/paris/lutece/plugins/forms/web/entrytype/EntryTypeDefaultDataService.java

```diff
@@ -212,6 +211,25 @@ public List<String> responseToStrings( FormQuestionResponse formQuestionResponse
 
         return listResponseValue;
     }
+    
+    private String getResponseValueForExport( Entry entry, Response response)
+    {
+    	String strResponseValue = null;
+    	IEntryTypeService entryTypeService = EntryTypeServiceManager.getEntryTypeService(entry);
+    	if (entryTypeService instanceof AbstractEntryTypeGeolocation)
+    	{
+    		if (response.getField( ) != null && IEntryTypeService.FIELD_ADDRESS.equals((response.getField().getCode())))
+    		{
+    			strResponseValue = response.getResponseValue( );
```

Comment:
Why not change `getResponseValueForExport` in `AbstractEntryTypeGeolocation` ? Changes to `EntryTypeDefaultDataService` would not be necessary